docs/js-client-sdk.eppojsclient.getassignment.md

```diff
@@ -4,7 +4,7 @@
 
 ## EppoJSClient.getAssignment() method
 
-<b>Signature:</b>
+**Signature:**

```

Comment:
The latest auto-generated docs use `**` for bolding and `_` for italic. (which I prefer in GitHub markdown anyways 📈 )